features/achordion.c

```diff
@@ -36,6 +37,14 @@ static uint16_t hold_timer = 0;
 // Eagerly applied mods, if any.
 static uint8_t eager_mods = 0;
 
+#if ACHORDION_TYPING_STREAK_TIMEOUT > 0
```

Comment:
It would be useful to have a per-key configurable streak timeout, based on which tap-hold key is active. How about use a symbol like "ACHORDION_STREAK" to enable the option and a "weak" attribute function like this, similar to how `achordion_timeout()` is implemented:

```c
#ifdef ACHORDION_STREAK
__attribute__((weak)) uint16_t achordion_streak_timeout(uint16_t tap_hold_keycode) {
  return 100;  // Default of 100 ms.
}
#endif
```

What do you think?

Reply:
I am not sure how that would work. I think this would required a different logic to set the future time instead of the time of the last key being pressed. I guess you would want to increase that timeout for the weaker finger but the current implementation only check the current key being pressed, so I don't think that would work well.

It's worth trying though.

Reply:
Ha, but with the new timer logic, that should work just fine. Ok I'll add that.

Reply:
Awesome! Thanks for adding that in!